core/lib/state/src/rocksdb/mod.rs

```diff
@@ -122,6 +122,14 @@ impl RocksdbStorage {
         }
     }
 
+    #[cfg(test)]
+    pub fn new_testing(path: &Path) -> Self {
+        let mut new_self = Self::new(path);
+        new_self.enable_enum_index_migration(100);
```

Comment:
`new()` also creates DB with `enum_index_migration_chunk_size=100`, so this method can be dropped